README.md

```diff
@@ -124,7 +124,7 @@ hardware or simulators to Qiskit. The backend class includes a common interface
 <!-- This is not an exhaustive list, and if you maintain a provider package please feel free to open a PR to add new providers -->
 
 You can refer to the documentation of these packages for further instructions
-on how to get access and use these systems.
+on how to get access to use these systems.
```

Comment:
I feel like the previous wording is more descriptive as it's distinguishing that getting access and using the systems are different tasks.